aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -1122,6 +1122,9 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   m.impl("unfold", unfold_batching_rule);
   m.impl("unsqueeze", unsqueeze_batching_rule);
   m.impl("view", view_batching_rule);
+  // From the perspective of vmap, view and _unsafe_view are the same.
+  // (they are only different w.r.t. autograd).
+  m.impl("_unsafe_view", view_batching_rule);
```

Comment:
cc @zou3519 I had to add a batching rule for `_unsafe_view` to get `test_ops_gradients.py` to pass (before it would use the fallback, but it can't anymore now that `_unsafe_view` is officially a view op)

I also ran `pytest test/test_ops.py test/test_vmap.py -k "reshape"` without error in the functorch repo. But I'll probably need to patch up functorch in any case to add the new batching rule, right?

Reply:
You will need to add a new batching rule in functorch. Also, vmap may run before autograd (this is the case for the vmap in core), so the batching rule should not turn _unsafe_view into a regular view, right?